lib/std/room.c

```diff
@@ -3,7 +3,8 @@
 inherit cont "/std/container";
 
 static mapping exits, hidden_exits, areas, items;
-static int last_exit, weather, light;
+static int last_exit, weather, light, pk;
```

Comment:
👍 

From time to time, I've thought about an arena where players can go to fight each other.